README.md

```diff
@@ -0,0 +1,52 @@
+# Difflint #
+
+Do you have a codebase with inconsistent code style, and you want to
```

Comment:
I disagree, since this is kind of a marketing document in my opinion. The README is the first thing that you look at when visiting the repository on Github, and in my opinion it should answer the question "why do I want to use this software" right off the bat.